client/src/components/Seat/index.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import './style.scss';
+
+const Seat = () => {
+  const [selected, setSelected] = useState(false);
+  // TO DO gest variables from props
+  const occupied = false;
+  const seatNr = 1;
+  // TO DO move the handle function to the parent component
+  const handleSeatSelected = (e) => {
+    e.target.classList.toggle('selected');
+    setSelected((prevSelected) => !prevSelected);
+  };
+  return (
+    <button
```

Comment:
Since we are selecting it, I am starting to wonder if it should not be input afterall. We will need to submit it forward as an array of tickets later somehow.